app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,28 @@
+class QuotesController < ApplicationController
+  def show
+    @quote = Quote.find params[:id]
```

Comment:
What behaviour would you expect here if you entered a url with an `id` that did not match any model in the database?